sdks/java/core/src/main/java/com/google/cloud/dataflow/sdk/transforms/windowing/Window.java

```diff
@@ -599,6 +600,33 @@ public void validate(PCollection<T> input) {
       return output.setWindowingStrategyInternal(outputStrategy);
     }
 
+    @Override
+    public void populateDisplayData(DisplayData.Builder builder) {
+      builder
+          .add("windowFn", windowFn.getClass())
+          .include(windowFn);
+
+      if (trigger != null) {
+        builder.add("trigger", trigger.toString());
```

Comment:
I've added the code to make these better. For the above example, it now formats as:

    AfterWatermark.pastEndOfWindow().withLateFirings(AfterProcessingTime.pastFirstElementInPane().plusDelayOf(10 minutes))

Until we are ready to tackle word breaking or some other way to represent structure, I think this should do.